frontend/javascripts/oxalis/view/right-border-tabs/scrollable_virtualized_tree.tsx

```diff
@@ -56,9 +58,15 @@ function ScrollableVirtualizedTree<T extends BasicDataNode>(
 
   return (
     <div ref={wrapperRef}>
-      <AntdTree {...props} onDragOver={onDragOver} />
+      <AntdTree {...props} onDragOver={onDragOver} ref={ref} />
```

Comment:
ref is now a separate arg and thus needs to be passed explicitly :man_shrugging: